plugins/basic/nexus-repository-httpbridge/src/main/java/org/sonatype/nexus/repository/httpbridge/internal/ViewServlet.java

```diff
@@ -121,7 +122,9 @@ protected void service(final HttpServletRequest httpRequest, final HttpServletRe
       log.debug("Service completed");
     }
     catch (Exception e) {
-      log.warn("Service failure", e);
+      if (!(e instanceof AuthorizationException)) {
+        log.warn("Service failure", e);
+      }
```

Comment:
Pls create a separate trivial PR for it and let's merge it within 15 minutes :smile: 
It bothers me too with Maven (makes NX logs unusable, as mvn will try-and-repeat all the authenticated requests, filling logs with trash).